src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -186,7 +186,10 @@ internal Silo(string name, SiloType siloType, ClusterConfiguration config, ILoca
 
             logger.Info(ErrorCode.SiloGcSetting, "Silo starting with GC settings: ServerGC={0} GCLatencyMode={1}", GCSettings.IsServerGC, Enum.GetName(typeof(GCLatencyMode), GCSettings.LatencyMode));
             if (!GCSettings.IsServerGC || !GCSettings.LatencyMode.Equals(GCLatencyMode.Batch))
```

Comment:
My recollection is that ServerGC means a heap per core and parallel GC. Both are meaningless on a single-core system. I suspect that's why the setting is effectively ignored by .NET.

My concern is that the warning on a single-core machine isn't actionable - what do we expect the user who reads it to do? If they can't do anything, why confuse them with the warning to begin with?